logging.yml

```diff
@@ -25,6 +25,16 @@ loggers:
     propagate: false
     handlers:
       - stdout-context
+  asyncpgsa.query:
+    level: ERROR
+    propagate: false
+    handlers:
+      - stdout
+  slack.io.abc:
```

Comment:
I'm not 100% sure this will catch everything, iirc `slack.io` would be better

Reply:
Sounds good, this was just the one showing more info than I was expecting (emails and our token, for example)